### PR TITLE
[9.0] [port] l10n_ch_bank

### PR DESCRIPTION
**Pull request by yvaucher**
Depends:
- [x] #179 

post_init_hook due to https://github.com/odoo/odoo/issues/10696

### CODE REVIEW

**Comment by philhug** [comment]
@yvaucher there seems to be a small bug in the bank.xml
Here's a patch:

diff --git a/l10n_ch_bank/data/bank.xml b/l10n_ch_bank/data/bank.xml
index 29a32fb..61215b1 100644
--- a/l10n_ch_bank/data/bank.xml
+++ b/l10n_ch_bank/data/bank.xml
@@ -86295,7 +86295,7 @@
         <field name="bank_postaccount">*19-6989-5</field>
         <field name="active" eval="True"/>
     </record>
-    <record model="res.bank" id="bank_80539_No entry found for host https://www.transifex.com. Creating...">
-    <record model="res.bank" id="bank_80539_0009">
       <field name="name">Raiffeisenbank Goms</field>
       <field name="clearing">80539</field>
       <field name="code">Raiffeisen</field>

**Comment by yvaucher** [comment]
@philhug thanks I see there is still some issues in ids due to my last commit

**Comment by yvaucher** [comment]
@philhug note that you can format your code in message by putting `back quotes` or

```
adding 4 spaces
at start of line
```

Your patch wasn't readable but I found the issue

This is now fixed

File: l10n_ch_bank/models/res_config.py [hunk 1]
```diff
@@ -1,25 +1,7 @@
...
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
-
+# © 2011-2014 Nicolas Bessi (Camptocamp SA)
+# © 2014 brain-tec AG
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 """
 Adds a button allowing to reload all the banks from the XML file.
 Thus, if this module has already been installed but the data is changed,
```
Comment (yvaucher) [inline l10n_ch_bank/models/res_config.py]: To fix this feature is not working anymore as loading data has changed

**Comment by yvaucher** [comment]
@BT-ojossen Hello could you please review this? I replaced the XML data by csv data. (imported with a post_init hook). Thus as we now have a csv file it can be imported throught the web interface using the CSV import tool. This should be enough to replace the use of the update button in settings. Can you confirm?

**Comment by guewen** [comment]
:+1:

**Comment by BT-ojossen** [comment]
:+1: seems to be fine to me

**Pull request merged**